src/controllers/hid/hidcontroller.cpp

```diff
@@ -137,6 +137,70 @@ int HidController::close() {
     return 0;
 }
 
+void HidController::processInputReport(int& bytesRead, const int& currentBufferIndex) {
```

Comment:
Can we copy both value by value? That makes the signature easier to read.

Reply:
Done!